nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -99,67 +93,60 @@ def create_activation_quantizer_insertion_command(target_point: TargetPoint,
         """
         Returns backend-specific quantizer insertion command.
 
+        :param nncf_graph: NNCFGraph to get input/output shapes for the target point.
         :param target_point: Target location for the correction.
         :param quantizer_config: QuantizerConfig instance for the current layer.
-        :param weight_tensor: weight tensor to calculate weight quantization parameters.
-        :param node: NNCFNode with the attributes.
+        :param statistics: MinMaxTensorStatistic to calculate activation quantization parameters.
         :return: Backend-specific TransformationCommand for the quantizer insertion operation.
         """
 
     @staticmethod
     @abstractmethod
-    def minmax_statistic_collector(use_abs_max: bool,
-                                   reduction_shape: ReductionShape,
+    def minmax_statistic_collector(nncf_graph: NNCFGraph,
+                                   target_point: TargetPoint,
+                                   quantizer_config: QuantizerConfig,
                                    num_samples: int = None) -> TensorStatisticCollectorBase:
         """
         Returns backend-specific min max statistic collector.
 
-        :param use_abs_max: Whether to use absolute maximum value or not.
-        :param reduction_shape: Channel axes for the statistics aggregation.
+        :param nncf_graph: NNCFGraph to get input/output shapes for the target point.
+        :param target_point: Target location for the correction.
+        :param quantizer_config: QuantizerConfig instance for the current layer.
         :param num_samples: Maximum number of samples to collect.
         :return: Backend-specific TensorStatisticCollectorBase for the statistics calculation.
         """
 
     @staticmethod
     @abstractmethod
-    def mean_minmax_statistic_collector(use_per_sample_stats: bool,
-                                        use_abs_max: bool,
-                                        reduction_shape: ReductionShape,
-                                        num_samples: int = None,
-                                        window_size: int = None) -> TensorStatisticCollectorBase:
+    def mean_minmax_statistic_collector(nncf_graph: NNCFGraph,
+                                        target_point: TargetPoint,
+                                        quantizer_config: QuantizerConfig,
+                                        use_per_sample_stats: bool,
+                                        num_samples: int = None) -> TensorStatisticCollectorBase:
         """
         Returns backend-specific min max statistic collector.
 
+        :param nncf_graph: NNCFGraph to get input/output shapes for the target point.
+        :param target_point: Target location for the correction.
+        :param quantizer_config: QuantizerConfig instance for the current layer.
         :param use_abs_max: Whether to use absolute maximum value or not.
```

Comment:
Should be changed to `use_per_sample_stats`?

Reply:
Done, thank you!